src/main/java/invtweaks/InvTweaks.java

```diff
@@ -103,6 +101,44 @@
     private String mostRecentComparison = "";
     private boolean debugTree = false;
 
+    // add hoe, fishing rod, and shears to ToolTypes now
+    // TODO: may be added to ToolType class in the future as their omission
+    //  seems like an oversight
+    public static class ToolTypeWrapper {
```

Comment:
As I said, the mutability of `ToolType.values` suggests they're purposely leaving the class open-ended. Plus, the only thing worthy of a Forge PR would be adding the hoe, fishing rod, and shears; the priority mapping and ToolType comparison is specific to InvTweaks.

I personally know at least a dozen people anxiously awaiting the return of InvTweaks for 1.13, so I thought I'd offer a helping hand by streamlining some necessary adaptations. It's your project, though: if my PR doesn't mesh with your vision then feel free to reject it, no hard feelings.

Reply:
> Plus, the only thing worthy of a Forge PR would be adding the hoe, fishing rod, and shears
Yes, those.

>the priority mapping and ToolType comparison is specific to InvTweaks.
And, given they're used in exactly one place (and chances are if things were being redone to use the forge stuff anyway the whole priority thing would end up reworked anyway), they don't really justify having a special wrapper class just for those.

In fact, if you pay attention to the way the forge ToolType is defined it will *always* give you exactly the same object for the same key, so simple `==` comparison should work without explicitly checking names.

> awaiting the return of InvTweaks for 1.13, 
Don't expect it anytime soon. Forge + MC changes in things directly related to *extremely* fundamental stuff (e.g., item flattening, all the coremod stuff, maybe trying to get the keypresses handled more properly...) mean I've been trying to figure out how to even start seriously porting everything for a while.